File: app/src/main/java/com/reteno/sample/fragments/FragmentForcePush.java
package com.reteno.sample.fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.reteno.sample.BaseFragment;
import com.reteno.sample.databinding.FragmentForcePushBinding;

public class FragmentForcePush extends BaseFragment {

    private FragmentForcePushBinding binding;

    public FragmentForcePush() {
        // Required empty public constructor
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        binding = FragmentForcePushBinding.inflate(getLayoutInflater(), container, false);
        return binding.getRoot();
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        initListeners();
    }

    private void initListeners() {
        binding.btnForcePush.setOnClickListener(v -> {
            getReteno().forcePushData();
        });
    }

}

File: app/src/main/java/com/reteno/sample/fragments/FragmentForcePush.kt
package com.reteno.sample.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.reteno.sample.BaseFragment
import com.reteno.sample.databinding.FragmentForcePushBinding

class FragmentForcePush : BaseFragment() {

    private var binding: FragmentForcePushBinding? = null

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        binding = FragmentForcePushBinding.inflate(layoutInflater, container, false)
        return binding!!.getRoot()
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        initListeners()
    }

    private fun initListeners() {
        binding!!.btnForcePush.setOnClickListener { reteno.forcePushData() }
    }
}
